modules/virt-infer-instancetype-preference.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-specifying-instance-preference.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-infer-instancetype-preference_{context}"]
+= Inferring an instance type or preference
+
+You can use the `--infer-instancetype` and `--infer-preference` flags to infer which instance type, preference, or both to use to define the workload sizing and runtime characteristics of a VM.
+
+Inferring instance types, preferences, or both is enabled by default, and the`inferFromVolumeFairlure` policy of the `inferFromVolume` attribute is set to `Ignore`. Any errors when inferring from the boot volume are ignored, and the VM is created with instancetype and preference left unset.
```

Comment:
Shouldn't this be the first sentence in this doc given it outlines the default behaviour of `virtctl create vm`?